### PR TITLE
 install sqlflow playground with single-user mode on Kubernetes

### LINKED ISSUE #2349: Setup SQLFlow with single-user mode on Kubernetes
Using `kubectl create -f install-sqlflow.yaml` to install SQLFlow playground with the single-user mode which includes:
1. a MySQL server.
1. an SQLFlow server with the Jupyter Notebook server.
1. Argo workflow controller.

Remove https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/run/workflow_mode.md

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed #2349

### CODE REVIEW

File: doc/run/k8s/install-sqlflow.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```
Comment (wangkuiyi) [inline doc/run/k8s/install-sqlflow.yaml]: Is YAML section to add a namespace? How about we add a comment on why we need to create a namespace?

File: doc/run/k8s/install-sqlflow.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+kind: Namespace
+metadata:
+  name: sqlflow
+
+---
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (wangkuiyi) [inline doc/run/k8s/install-sqlflow.yaml]: Do we need to distinguish single-user or multiple-user mode in writing Kubernetes YAML fie? Or, can we use one YAML for deploying everything on minikube and Google Cloud and Alibaba Cloud?

**Review by wangkuiyi — COMMENTED** [review]
I am not familiar enough with Kubernetes deployment -- I am wondering can we write a YAML file for all kinds of Kubernetes cluster -- including minikube, Google Cloud, and Alibaba Cloud?

File: doc/run/k8s/install-sqlflow.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      labels:
+        app: sqlflow-mysql
+    spec:
+      volumes: 
+      containers:
+      - image: sqlflow/sqlflow:mysql
+        name: mysql
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3306
```
Comment (lhw362950217) [inline doc/run/k8s/install-sqlflow.yaml:46]: It seems that we have to change the `start.sh` in SQLFlow image to access this DB.

File: doc/run/k8s/install-sqlflow.yaml (hunk 1 above)
Comment (Yancey0623) [reply doc/run/k8s/install-sqlflow.yaml]: A namespace provides a scope for Kubernetes resource e.g. Pod, Service, but thanks for your reminding, I think we don't need a namespace `sqlflow` Here, SQLFlow can run on the default namespace.

File: doc/run/k8s/install-sqlflow.yaml (hunk 2 above)
Comment (Yancey0623) [reply doc/run/k8s/install-sqlflow.yaml]: I think we should do this distinguish.
For SQLFlow developers, he/she can deploy SQLFlow with the single-user mode on Kubernetes, the multi-users model is no use for he/she.
For an SQLFlow playground with multi-user, we should deploy SQLFlow with the multi-users mode on Kubernetes.

**Comment by Yancey0623** [comment]
> I am not familiar enough with Kubernetes deployment -- I am wondering can we write a YAML file for all kinds of Kubernetes cluster -- including minikube, Google Cloud, and Alibaba Cloud?

Yes, all the kubernetes can use one YAML file unless not compatible with open-source Kubernetes YAML e.g. internal customed Kubernetes cluster.

File: doc/run/k8s/install-sqlflow.yaml (hunk 3 above)
Comment (Yancey0623) [reply doc/run/k8s/install-sqlflow.yaml:46]: Thanks, will use a specified command line instead of the default `start.sh`

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Pull request merged**